### ISSUE TITLE
Open Drawer with Swipe from Edge

### ISSUE THREAD

**Issue by asndev** [root-two/react-native-drawer#167]
hi,

is it possible to open the drawer by swiping from the edge of the screen?

Eg:
[example1](https://camo.githubusercontent.com/aa4203d407bc63d3b04a08a2bcbc3b706397984a/687474703a2f2f6f6936312e74696e797069632e636f6d2f326e396c32647a2e6a7067)
(https://github.com/react-native-fellowship/react-native-side-menu)

[example2](https://raw.githubusercontent.com/iodine/react-native-drawer-layout/master/example.gif)
(https://github.com/iodine/react-native-drawer-layout)

Cheers,

**Comment by rt2zz** [comment]
Yes that should work out of the box. Are you having problems with this?

**Comment by asndev** [comment]
yes, I'm going to evaluate and test this. You can close this issue until I have a reproducable scenario.

**Comment by robertodias180** [comment]
i solved it with this `panOpenMask={0.1}`

**Comment by jackmead515** [comment]
Hi, know this is an old issue but it hasn't been resolved. What is the prop for this?
